MrTheGood/projectb_android
# Onboarding

### Changes:
 * Added some cool red themes because why tf not
 * Added onboarding page that is shown on first startup
 * Onboarding can be restarted by a button in the settings page
 * Some code cleanup

### How to test: 
 1. **Test** the Light Red and Dark Red themes
 2. **Confirm** if the onboarding page is correctly shown on first run
 3. **Confirm** that it does not show up after skip or finish
 4. **Confirm** that it looks how it should look on **your phone**

### Screenshots:
**Onboarding page**
![device-2019-05-22-223342](https://user-images.githubusercontent.com/23239376/58207281-b0ab9a00-7ce2-11e9-8ad0-e0dd6618c03c.png)
![device-2019-05-22-223350](https://user-images.githubusercontent.com/23239376/58207283-b0ab9a00-7ce2-11e9-93c5-6d294fb4b3bf.png)
**Light Red theme**
![device-2019-05-22-223430](https://user-images.githubusercontent.com/23239376/58207284-b0ab9a00-7ce2-11e9-92d2-1b5bb7a26646.png)
**Dark Red theme**
![device-2019-05-22-223442](https://user-images.githubusercontent.com/23239376/58207285-b0ab9a00-7ce2-11e9-8ca3-6ec3ccd9d959.png)
**Onboarding page in Dark Red theme**
![device-2019-05-22-223549](https://user-images.githubusercontent.com/23239376/58207286-b1443080-7ce2-11e9-8792-d949aa72792e.png)
![device-2019-05-22-223555](https://user-images.githubusercontent.com/23239376/58207288-b1443080-7ce2-11e9-9032-ba1abdf3dc2d.png)
